jenkins/debian/debos/scripts/buster-crostests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# Build-depends needed to build the test suites, they'll be removed later
+BUILD_DEPS="\
+    git \
+    python3-setuptools \
+    python3-pip \
+"
+
+apt-get install --no-install-recommends -y ${BUILD_DEPS}
+
+########################################################################
+# Build and install tests                                              #
+########################################################################
+
+pip3 install git+https://gitlab.collabora.com/chromiumos/crostests.git
```

Comment:
Maybe this should have been added to the existing `https://gitlab.collabora.com/chromium` group?

Reply:
OK it's on kernel.org now, even better :)